### PR TITLE
Fix ICDS reconciliation task and improve stagnant case task.

### PR DESCRIPTION
**Pull request by emord**
##### SUMMARY
Fixes a couple of recent tasks for ICDS.

I also saw some bad performance by one of the stagnant case queries. Including the doc_id and not including a <= inserted_at filter was causing the query to do a full sequential scan for reasons i haven't fully understood yet.

### CODE REVIEW

File: custom/icds_reports/models/util.py [hunk 1]
```diff
@@ -164,14 +164,14 @@ def setup_days_records(cls, day):
...
-                'tasks': ['static-tasks_cases'],
-                'tech_issue': ['static-tech_issue_cases'],
+                'static-commcare_user_cases': ['commcare-user'],
+                'static-ccs_record_cases': ['ccs_record'],
+                'static-child_health_cases': ['child_health'],
+                'static-hardware_cases': ['hardware'],
+                'static-household_cases': ['household'],
+                'static-person_cases_v3': ['person'],
+                'static-tasks_cases': ['tasks'],
+                'static-tech_issue_cases': ['tech_issue'],
```
Comment (emord) [inline custom/icds_reports/models/util.py]: I already fixed these in the models already created and will fix the other ones as they are created

File: custom/icds_reports/tasks.py [hunk 2]
```diff
@@ -747,7 +747,8 @@ def _find_stagnant_cases(adapter, latest_datetime):
     query_object = adapter.get_query_object()
     return (
         query_object.with_entities(table.c.doc_id, table.c.inserted_at).filter(
-            table.c.inserted_at >= latest_datetime
+            table.c.inserted_at >= latest_datetime,
+            table.c.inserted_at <= datetime.utcnow()  # This filter is used to force postgres to use the index
```
Comment (mkangia) [inline custom/icds_reports/tasks.py]: is this something that is needed each time we filter on a datetime format index?

File: custom/icds_reports/tasks.py [hunk 3]
```diff
@@ -748,7 +748,7 @@ def _find_stagnant_cases(adapter, latest_datetime):
     return (
         query_object.with_entities(table.c.doc_id, table.c.inserted_at).filter(
             table.c.inserted_at >= latest_datetime
-        ).distinct().order_by(table.c.inserted_at, table.c.doc_id)[:BATCH_SIZE]
+        ).distinct().order_by(table.c.inserted_at)[:BATCH_SIZE]
```
Comment (mkangia) [inline custom/icds_reports/tasks.py]: I don't fully understand the connection between this change and the commit description.

**Review by mkangia — COMMENTED** [review]
Don't have enough context on this to review it properly, but tried to review as much as i could.

**Review by snopoke — APPROVED** [review]

File: custom/icds_reports/tasks.py (hunk 2 above)
Comment (emord) [reply custom/icds_reports/tasks.py]: honestly I'm not sure. in investigating the agg querires,  we've recently noticed some issues with our postgres statistics being "off" or at least not producing the queries that we expect. Its possible that this is just hitting some strange edge case with postgres query planner

**Pull request merged**